src/pages/account/CreateNewUser.tsx

```diff
@@ -208,10 +211,15 @@ const CreateNewUser = () => {
                 </Button>
                 <Button
                   disabled={
-                    !validator.isEmail(newUser.email!) ||
-                    validator.isEmpty(newUser.last_name!) ||
-                    validator.isEmpty(newUser.first_name!) ||
-                    validator.isEmpty(newUser.role!)
+                    newUser.email === undefined ||
```

Comment:
can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) and [null coalesce operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) to make this less verbose

Reply:
Edited!